dds_web/api/unit.py

```diff
@@ -59,3 +62,21 @@ def get(self):
                 "Contact Email",
             ],
         }
+
+
+class MOTD(flask_restful.Resource):
+    """Add a new MOTD message."""
+
+    @auth.login_required(role=["Super Admin"])
+    @logging_bind_request
```

Comment:
```suggestion
    @logging_bind_request
    @json_required
```
Also import from the dds_decorators